osu.Game.Rulesets.Osu/Mods/OsuModTarget.cs

```diff
@@ -189,21 +185,19 @@ public override void ApplyToBeatmap(IBeatmap beatmap)
                                  return (OsuHitObject)newCircle;
                              }).ToList();
 
-            addHitSamples(hitObjects);
+            addHitSamples(originalHitObjects, hitObjects);
```

Comment:
For better legibility, when a method's purpose is to apply changes to a certain object, that object should be the first parameter.